geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -245,8 +268,17 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.unlisten(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        if (goog.isDefAndNotNull(feature.get('__source__')) &&
+            feature.get('__source__') == 'mymaps') {
```

Comment:
You don't need to test that `feature.get('__source__')` is defined and not null here. The following should work:

```js
if (feature.get('__source__') === 'mymaps') {
  // …
}
```